Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp

```diff
@@ -103,7 +103,7 @@ ThreadedCompositor::ThreadedCompositor(Client& client, ThreadedDisplayRefreshMon
         m_display.updateTimer->startOneShot(Seconds { 1.0 / m_display.displayUpdate.updatesPerSecond });
 #endif
 
-        m_scene = adoptRef(new CoordinatedGraphicsScene(this));
+        m_scene = adoptRef(new CoordinatedGraphicsScene(this, m_client.settings().propagateDamagingInformation()));
```

Comment:
I think we should pass the setting directly as construct parameter instead of exposing all the settings in the ThreadedCompositor client